CovertInfiltration/Localization/XComGame.int

```diff
@@ -281,8 +281,8 @@ FacilityName="Living Quarters"
 Summary="Increases Barracks capacity by a further 15, allowing more soldiers to be maintained without an increase to recovery times."
 
 [Reward_Rumor X2RewardTemplate]
-DisplayName="Resistance Rumor"
-RewardDetails="Reveal a Point of Interest on the Geoscape"
+DisplayName="Intel and Rumor"
+RewardDetails="Gain Intel and reveal a Point of Interest on the Geoscape"
```

Comment:
I think the loc should mention that it's "a bit of intel" or something

Reply:
I don't actually think the Reward Details string ever shows up anywhere.

Reply:
Well, then `DisplayName` should somehow indicate it

Reply:
It does, the display name is combined with the intel amount on the mission UI. So it reads as `X Intel and Rumor`

Reply:
Oh ok

Reply:
Something to consider: the UI supports multiple rewards. What is the intel and rumour were separated into individual rewards?